src/main/java/Database/playlistLibrary.java

```diff
@@ -46,9 +45,10 @@ private Map<Integer, playlistDsData> loadFile() {
                 // assumes 5 columns, properly filled
                 String line = in.nextLine();
                 String[] data = line.split(",");
-                int id = Integer.parseInt(data[0]);
-                saved.add(id);
-                p.put(Integer.parseInt(data[0]), new playlistDsData(data));
+                if (data.length == 5) {
+                    int id = Integer.parseInt(data[0]);
+                    p.put(id, new playlistDsData(data));
+                }
```

Comment:
Should there be some kind of warning of result if the 5 columns are not properly filled and you don't complete loading the file?

Reply:
It just skips the line without 5 columns and moves on, so it should be fine. Theoretically, my database file is the only thing that should be touching the csv files so I think it's an ok assumption.